tools/eos-content-merge

```diff
@@ -4,12 +4,14 @@
 from collections import OrderedDict
 from configparser import ConfigParser
 import json
+from libeosshellcontent import ShellContent
 import os
 import re
 import shutil
 import sys
 
-CONTENT_FILE = '/usr/share/eos-shell-content/content.json'
+CONTENT_DIR = os.getcwd()
+CONTENT_FILE = CONTENT_DIR + '/content/Default/apps/content.json'
```

Comment:
Hmm, sorry. I did this as a test but since I finished this task only a few days after it I completely forgot I had done it (and missed it when I added the patch). It was not my intention to propose this.